### PR TITLE
advanced deployment options

### PR DESCRIPTION
**Pull request by matthewfincher**

### CODE REVIEW

File: config/accountKeyFile.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "key": "b795ff14bd0198db79dad7d6df2e1103", "iv": "3bd7ba6ad18b719bd1d7cc08e188ae55", "createdAt": "2016-11-05T00:00:59.300Z" }
```
Comment (owaxh) [inline config/accountKeyFile.json]: What is the account key file for?

File: config/env/default.js [hunk 2]
```diff
@@ -33,6 +33,8 @@ module.exports = {
   authStrategy: 'local',
   // authStrategy: 'ldap',
 
+  caBundlePath: '../sslcerts/cacert.pem',
```
Comment (owaxh) [inline config/env/default.js]: If we don't specific a key - will it work with the current bundle?

File: modules/landscapes/client/controllers/admin.client.controller.js [hunk 3]
```diff
@@ -14,17 +14,18 @@
         var vm = this;
 
         vm.menu = [
-            'Users',
-            'Roles',
-            'Groups',
+            // 'Users',
```
Comment (owaxh) [inline modules/landscapes/client/controllers/admin.client.controller.js]: why comment out?

File: config/env/default.js (hunk 2 above)
Comment (matthewfincher) [inline config/env/default.js]: Yes, it work with the current bundle. This is a forward looking configuration parameter that is not being used for now. The CA Bundle Path can be changed at the Account level or at deploy time.

File: modules/landscapes/client/controllers/admin.client.controller.js (hunk 3 above)
Comment (matthewfincher) [inline modules/landscapes/client/controllers/admin.client.controller.js]: I broke out the "Users, Roles and Groups" management capabilities into a new menu called "User Admin" with the intention of providing more granular access controls: System Admin vs User Admin.

**Pull request merged**